depends/packages/rapidcheck.mk

```diff
@@ -1,18 +1,17 @@
 package=rapidcheck
-$(package)_version=10fc0cb
-$(package)_download_path=https://github.com/MarcoFalke/rapidcheck/archive
-$(package)_file_name=$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=9640926223c00af45bce4c7df8b756b5458a89b2ba74cfe3e404467f13ce26df
+$(package)_version=3eb9b4ff69f4ff2d9932e8f852c2b2a61d7c20d3
+$(package)_download_path=https://github.com/emil-e/rapidcheck/archive
```

Comment:
Is there a reason we don't use the bitcoin core fork? https://github.com/bitcoin-core/rapidcheck

Reply:
We haven't actually made any changes yet, so I think sticking with upstream is best for now.